test/com/google/javascript/jscomp/PolymerPassTest.java

```diff
@@ -184,6 +186,19 @@ public void testVarTarget() {
             "X = Polymer(/** @lends {X.prototype} */ {",
             "  is: 'x-element',",
             "});"));
+
+    disableTypeCheck();
```

Comment:
Add a comment that the type checker doesn't understand ES6 and to remove when it does